README.md

```diff
@@ -165,15 +154,17 @@ check                          0
 ```
 Date       Description             Accounts            
 -------------------------------------------------------------------------------
-11/4/2019  weekly groceries        grocery                   455.00      455.00
-                                   credit_card_amex         -455.00        0.00
-11/4/2019  raspberry pi            general                    50.00       50.00
-                                   credit_card_amex          -50.00        0.00
-05/23/2020 business stuff          cash_checking             600.00      600.00
-                                   general                  -600.00           0
-06/21/2020 grocery store           general                    20.00       20.00
-                                   grocery                   180.00      200.00
-                                   credit_card_amex         -200.00           0
+11/4/2019  weekly groceries        expense:grocery           455.00      455.00
+                                   expense:grocery          -455.00       0.00
+07/04/2020 mortage                 expense:mortgage         2000.00     2000.00
+                                   expense:mortgage        -2000.00       0.00
+07/04/2020 stuff                   expense:general          1000.00     1000.00
+                                   asset:cash_savings      -1000.00        0.00
+                                                              -0.00           0
+06/21/2020 grocery store           expense:general            20.00       20.00
+                                   expense:grocery           180.00      200.00
+                                   asset:cash_checking      -200.00        0.00
+                                                              -0.00           0
```

Comment:
This is a bit different that What I would have expected. Possibly a redundant line in the output. The line above shows the balancing down to `0.00` which is what I would expect.

Reply:
this should be fixed now - good catch!

Reply:
Nice. Looks good.